user_interaction/account_pages/config.py

```diff
@@ -1,20 +1,22 @@
 from django.urls import path
 
-from user_interaction.config import AccountConfig
+from user_interaction.accountcollective import AccountBaseConfig
 
 from .views import SiteAccountView, AccountPasswordChangeView, LayoutPreferencesUpdateView
 
 
-class TestAccountConfig(AccountConfig):
+class AccountSettingsConfig(AccountBaseConfig):
     url_keyword = 'site'
-    tab_select_keyword = 'tab_account_info'
     name = 'Account'
     url_name = 'site_account'
+    order_value = 100  # Value determining the order of the tabs on the Account page
```

Comment:
I'd suggest to use the following ordering on the account tabs: Account, Membership, Groups, Inventory, Achievements